src/test/java/seedu/anilist/logic/parser/RenameCommandParserTest.java

```diff
@@ -36,11 +37,9 @@ public void parse_missingParts_failure() {
 
     @Test
     public void parse_invalidPreamble_failure() {
-        // negative index
-        assertParseFailure(parser, "-5" + NAME_DESC_AKIRA, MESSAGE_INVALID_FORMAT);
-
-        // zero index
-        assertParseFailure(parser, "0" + NAME_DESC_AKIRA, MESSAGE_INVALID_FORMAT);
+        // not a valid number
+        assertParseFailure(parser, "3.141582653" + NAME_DESC_AKIRA, MESSAGE_INVALID_FORMAT);
```

Comment:
3.1415926535 is rounded off to 3.141592654

Reply:
Fair enough lol